src/js/index.js

```diff
@@ -85,7 +104,12 @@ function cascade(force = false) {
 	}
 }
 
-window.addEventListener('load', () => {
+window.addEventListener('DOMContentLoaded', () => {
+	lazyLoadStyleSheet('/assets/css/index.css');
```

Comment:
This is the path to the busted assets only in dev mode. It might be changed one layer upper by netlify also so we shouldn't reference it directly like this.

Reply:
Will the path always be `assets/...`? Maybe using `location.hostname` will solve it, otherwise I can make a fake `link` with a `data-href` attribute which will be swapped when the page is ready :)

Reply:
I'd go with `data-href` on the `link` so we keep our dynamic filenames in the html (`bustedAssets.css` variable).